backend/modules/auth/router.py

```
@@ -0,0 +1,654 @@
+        allow_contact=user.allow_contact
+    )
+    
+    # Create access token and set HttpOnly cookie
+    access_token = auth_service.create_access_token(data={"sub": str(db_user.id)})
+    auth_service.set_auth_cookie(response, access_token)
+    
+    return db_user
+
+
+@router.post("/login", response_model=UserLoginResponse)
+async def login_user(user: UserLogin, response: Response, db: Session = Depends(get_db)):
```

Ruff (0.14.5)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)